docs/openapi.json

```diff
@@ -3140,6 +3166,43 @@
                     }
                 }
             },
+            "StakerResponse": {
+                "description": "A successful response with the total and current number of staked tokens, as well as the staked number of nodes staked and chips owned by the specific staker.",
+                "content": {
+                    "application/json": {
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "address": {
+                                    "type": "string",
+                                    "description": "The wallet address of the staker.",
+                                    "example": "0x69982e017acc0fde3d1542205089a8d3eafcd1b7"
+                                },
+                                "total_staked_tokens": {
+                                    "type": "string",
+                                    "description": "Total cost of chips owned by the staker.",
```

Comment:
> Total cost of chips owned by the staker

this is really a confusing description, I suggest: Total number of tokens historically staked (including unstaked tokens).